### ISSUE TITLE
Update existing Pull requests

### ISSUE THREAD

**Issue by martinheidegger** [greenkeeperio/greenkeeper#135]
Usually developers seem to push out a patch release right after a major release because they found something little broken. In the case of `standard` the developer release `6.0.0` ... `6.0.4` in rapid succession and in all projects that i have I get 5 new Pull Requests. It would be nice if greenkeeper would (maybe optional) automatically update Pull Requests if they are for the same dependency. It would be even okay for me to combine Pull requests over different packages where the name of the Pull Request would be updated from "Update Standard to version 6.0.1" to "Update several dependencies". I realise that this might not be desired for everyone which is why a flag would be awesome.

**Comment by boennemann** [comment]
Hello @martinheidegger,

thanks a lot for your feedback, we're definitely seeing this as a problem.
You can read up on the full history of our decision and the following feature addition in this issue: #62

tl;dr On merge we're closing all Greenkeeper pull-requests for the same or lower version range of the same dependency. If this still annoys you then please ask the authors of your packages to use distribution channels as described in this blogpost: https://medium.com/greenkeeper-blog/one-simple-trick-for-javascript-package-maintainers-to-avoid-breaking-their-user-s-software-and-to-6edf06dc5617#.mqv2tkveb

Best,
Stephan
